### ISSUE TITLE
Net2Cog should not default to Web Mercator projection if CRS is not explicitly set.

### ISSUE THREAD

**Issue by autydp** [podaac/net2cog#12]
I recommend removing the code that sets CRS to Web Mercator in the absence of any CRS in the source data.
- HyBIG will reproject if necessary.
- Most often, the absence of an explicit CRS implies Geographic (LatLon) projection, not Web-Mercator.
- RioXarray (CoGeo? - Cloud Optimized GeoTIFF package) interprets correctly (as Geographic) without CRS.

**Comment by autydp** [comment]
Note - if the original intention was for PO.DAAC's browse-image generation to always produce Web-Mercator images, I might suggest this be moved to the chaining implementation that calls HyBIG, or even the external invocation of that chain - where the CRS can be asserted in the request - rather than being burned into the Net2Cog implementation.

**Comment by frankinspace** [comment]
I agree, we can remove the code that sets CRS to Web Mercator in the absence of any CRS in the source data.

I think that made it in originally as I was messing with settings to try to get the image to display correctly (not-inverted https://github.com/podaac/net2cog/issues/7, which I'm coming to understand is not actually a problem).
